templates/bridgeserver2.yaml

```diff
@@ -117,6 +114,23 @@ Resources:
         - Namespace: 'aws:autoscaling:updatepolicy:rollingupdate'
           OptionName: RollingUpdateType
           Value: 'Time'
+        - Namespace: 'aws:ec2:vpc'
+          OptionName: ELBSubnets
+          Value: !Join
+            - ','
+            - - !ImportValue us-east-1-BridgeServer2-vpc-PublicSubnet
+              - !ImportValue us-east-1-BridgeServer2-vpc-PublicSubnet1
+              - !ImportValue us-east-1-BridgeServer2-vpc-PublicSubnet2
+        - Namespace: 'aws:ec2:vpc'
+          OptionName: Subnets
+          Value: !Join
+            - ','
+            - - !ImportValue us-east-1-BridgeServer2-vpc-PrivateSubnet
+              - !ImportValue us-east-1-BridgeServer2-vpc-PrivateSubnet1
+              - !ImportValue us-east-1-BridgeServer2-vpc-PrivateSubnet2
+        - Namespace: 'aws:ec2:vpc'
+          OptionName: VPCId
+          Value: !ImportValue us-east-1-BridgeServer2-vpc-VPCId
```

Comment:
changes look good.  I would suggest doing this either in this change or a follow up change: 

should add:
```
        - Namespace: 'aws:ec2:vpc'
          OptionName: AssociatePublicIpAddress
          Value: 'false'

        - Namespace: 'aws:autoscaling:launchconfiguration'
          OptionName: EC2KeyName
          Value: !Ref EC2KeyName
```

should parameterize the values:
```
        - Namespace: 'aws:autoscaling:updatepolicy:rollingupdate'
          OptionName: RollingUpdateType
          Value: 'Time'


        - Namespace: 'aws:elasticbeanstalk:command'
          OptionName: DeploymentPolicy
          Value: 'Rolling'
```

should remove:
```
        - Namespace: 'aws:elasticbeanstalk:application:environment'
          OptionName: APP_DEPLOY_BUCKET
          Value: !Ref AppDeployBucket
```

Reply:
Added the AssociatePublicIpAddress field.

Parameterized the Rolling deployment settings, with default values and allowed values.

Removed APP_DEPLOY_BUCKET from elastic beanstalk env vars.

I didn't add the EC2 key, since we no longer need to SSH into the elastic beanstalk hosts. If we need to in the future, we can do the work then.